go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/Shopify/sarama v1.27.2
 	github.com/blevesearch/bleve v1.0.13
 	github.com/dgraph-io/badger/v4 v4.1.0
-	github.com/dgraph-io/dgo/v230 v230.0.1-rc1
```

Comment:
Should this be done as part of this PR? @mangalaman93

Reply:
I don't mind, do you think we should have a new PR?